packages/webdriver-image-comparison/src/methods/screenshots.test.ts

```diff
@@ -375,12 +340,13 @@ describe('screenshots', () => {
             const options: FullPageScreenshotDataOptions = {
                 addressBarShadowPadding: 6,
                 devicePixelRatio: 2,
+                // @ts-expect-error
```

Comment:
why the expect error here? I suggest we fix the issue instead

Reply:
Will fix it, thanks